gae/room/ndb_models.py

```diff
@@ -1167,6 +1167,7 @@ def _GetRateFromArray(default, array, activity_date):
 
 class StaffPosition(SearchableModel):
   """Staff positions that have hourly billing."""
+  name = ndb.StringProperty()
```

Comment:
StaffPosition needed a "name" attribute for sorting.
Because I used staff.__EntryList_  helper method to create a list view. 
Will I need to create a more custom method?

Reply:
Yes you can create a name method using @property which will work.  See examples in this file.